modules/storage/local.go

```diff
@@ -107,8 +119,19 @@ func (l *LocalStorage) Stat(path string) (os.FileInfo, error) {
 	return os.Stat(filepath.Join(l.dir, path))
 }
 
+func isLocalPathValid(path string) bool {
+	a := filepath.Clean(path)
+	if strings.HasPrefix(a, fmt.Sprintf("..%c", filepath.Separator)) {
+		return false
+	}
+	return a == path
```

Comment:
I'm not exactly sure which kind of paths that will come in here but `filepath` will replace all slashes's with the os's seperater so it's possible that this could return false when it's actually true, just because the slashes differ.

Reply:
It will not replace slashes, see the test in my recent commit.